### DISCUSSION TITLE
Suggestion for FAQ: Dealing with old Intel Mac Homebrew on an M1 Mac

### DISCUSSION THREAD

**Discussion by sbeaumont** [Homebrew/discussions#3247] [Getting started]
Hi,

I thought I'd share this because it took me a while to figure out, and hopefully others may have use of this.

This deals specifically with the case if you have transferred from an Intel Mac to an Apple Silicon Mac with an existing Homebrew install. In my case I opted for it because having to reinstall everything wasn't worth it to me: I don't develop software that often anymore. So far that worked out for me because reinstalling Intel apps has been pretty painless for those cases I needed it.

However I encountered a bigger issue because my `pyenv install <version>`failed with an (openssl can't be found) build error.

After some digging I found that even though I had reinstalled Homebrew, there was still the existing version in /usr/local for Intel Mac hiding the /opt/homebrew version. By following the uninstall steps here:

https://apple.stackexchange.com/questions/410825/apple-silicon-port-all-homebrew-packages-under-usr-local-opt-to-opt-homebrew

and adding /opt/homebrew/bin to the front of my PATH, everything worked fine again.

Hope that helps someone in the same situation. Maybe a general FAQ answer "how to survive a move from Intel Mac to M1 if you were stupid enough to just do a straight copy of your old machine" would be good to add with the above tips..." :-)

**Comment by carlocab** [comment]
Thanks for the suggestion. Pull requests to improve our documentation welcome. You can find the docs hosted on https://brew.sh here: https://github.com/Homebrew/brew/tree/master/docs
